adaptors/library/jobs/satusehat-create-medication.js

```diff
@@ -0,0 +1,115 @@
+// Create a medication from visit data and lookup table
+
+fn(state => {
+  // state.visit is a Commcare Visit record
+  const properties = state.visit.properties;
+  const medicineKeys = extractMedicineKeys(properties);
+
+  const obatList = state.lookup_table.obat;
+
+  state.medicationSatusehatId = [];
+
+  // Map medication data for each relevant key
+  state.medicationMappedData = mapMedications(
```

Comment:
Still not sure about the `MappedData` suffix. I can see it's a repeated pattern so if it's clear for other users, then ignore me :)